roles/openshift_docker_registry/tasks/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+
+# TODO: Centralized openshift_config openshift_facts??
+# TODO: anchor this on an infrastructure node, default project
+- name: Query for OpenShift Docker Registry
+  command: openshift admin registry --credentials={{ openshift_config }}
+  register: registry_results
+  changed_when: False
```

Comment:
"service exists" is the only consistent method I've found with the last several releases.